src/components/BulkEditList/BulkEditListFilters/BulkEditListFilters.js

```diff
@@ -210,17 +214,25 @@ export const BulkEditListFilters = ({
     return (
       <>
         <Button
-          buttonStyle={criteria === CRITERIES.IDENTIFIER ? 'primary' : 'default'}
+          buttonStyle={isIdentifier ? 'primary' : 'default'}
           onClick={() => setFilters(prev => ({ ...prev, criteria: 'identifier' }))}
         >
           <FormattedMessage id="ui-bulk-edit.list.filters.identifier" />
         </Button>
         <Button
-          buttonStyle={criteria === CRITERIES.QUERY ? 'primary' : 'default'}
+          buttonStyle={isQuery ? 'primary' : 'default'}
           onClick={() => setFilters(prev => ({ ...prev, criteria: 'query' }))}
         >
           <FormattedMessage id="ui-bulk-edit.list.filters.query" />
         </Button>
+        {hasLogViewPerms &&
+        <Button
+          buttonStyle={isLogs ? 'primary' : 'default'}
+          onClick={() => setFilters(prev => ({ ...prev, criteria: 'logs' }))}
+        >
+          <FormattedMessage id="ui-bulk-edit.list.filters.logs" />
+        </Button>
```

Comment:
Petty: `<Button>` should be indented here. It's weird to me that `yarn lint` didn't catch this. Please review the `.eslintrc` settings for this repository and make sure there is no `rules` key overriding the defaults configured by `@folio/eslint-config-stripes`. If `rules` is present and you think the rule is worthwhile, open a PR in https://github.com/folio-org/eslint-config-stripes and put it on the agenda at Slack#stripes-architecture to discuss with a wider audience. If it's accepted, we all benefit from the better rule! If not, then you'll have some clean up to do. 

It sounds petty, but consistency is important in a big project like this. Letting each repo have its own pet rules makes it much harder to do work across multiple repos.